.c9/runners/pru0c.run

```diff
@@ -4,12 +4,13 @@
     "cmd" : [
         "make",
         "-f",
-        "/var/lib/cloud9/extras/pru/Makefile",
```

Comment:
This means that someone MUST create a new Makefile to add programs in their own directory. Can we keep that optional?

Reply:
Cloud9 has two built in runners for c files.  One uses a makefile and the other appears to just compile the file.  Is that optional enough?

Reply:
Don't see how that helps. The default runner should "just work", even if there is not a Makefile in the directory.

Why do you feel so strongly that a local Makefile must be included?

What about something like the Linux kernel Makefile structure:
* https://www.kernel.org/doc/Documentation/kbuild/makefiles.txt
* http://www.tldp.org/LDP/lkmpg/2.6/html/lkmpg.html#AEN189

The idea would be to use -C instead of -f such that a local Makefile would still be included to define additional variables. You can further include the 'all' definition to keep a 'make' execution the the directory to include the proper Makefile.

I'm just very opposed to making the Makefile required in each directory and further duplicating lines of Makefile that we are likely going to need to extend/update. If the Makefile was simple boilerplate stuff, copying it could be most effective (though if it is possible to consolidate it is better), but for complex stuff, it really matters to have it in one place.

We might want to move it to a /usr/share or /usr/lib folder if we want to use it more generally.